npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -146,6 +145,10 @@ func (pMgr *PolicyManager) reconcileChains(stopChannel <-chan struct{}) {
 
 // this function has a direct comparison in NPM v1 iptables manager (iptm.go)
 func (pMgr *PolicyManager) runIPTablesCommand(operationFlag string, args ...string) (int, error) {
+	return pMgr.runIPTablesCommandAndIgnoreErrorCode(-1, operationFlag, args...)
+}
+
+func (pMgr *PolicyManager) runIPTablesCommandAndIgnoreErrorCode(errCodeToIgnore int, operationFlag string, args ...string) (int, error) {
```

Comment:
It is a little hard to follow which operations are ok to ignore which errors.
Do we delegate error handling in a caller side?

We just have one `run` function and a caller will do right actions bases on return values.
It will be easier to follow up codes.

Reply:
The main consideration for ignoring an error code in this function is that this function logs when there's an error, but I think it shouldn't log an error if we ignore the error later. We previously had some of that code baked into run() for check operations, but I think we should also do it for deleting a rule.

Reply:
these changes will be moved to the next PR which updates policy/chain management